docs/source/gbd2017_models/causes/neoplasms/multiple_myeloma/multiple_myeloma_model.rst

```diff
@@ -7,9 +7,19 @@ Multiple Myeloma
 Disease Overview
 ----------------
 
-.. todo::
-
-   Add definition of multiple myeloma. 
+Multiple myeloma (MM) is a clonal plasma cell neoplasm with substantial morbidity and mortality, characterized by end organ damage—renal 
```

Comment:
This overview looks great. I recommend adding a little more information about how the disease is defined, if you find any additional information about that. The first sentence in this may not be easily interpretable to all. I also recommend adding more information about the prevalence or incidence of this disease for GBD 2017. It could be helpful to answer these questions: Is this a disease that is a leading cause of death or disability for any specific age groups or sexes? Does this disease affect specific regions/countries disproportionately? Is there any information in external literature that differs from how GBD defines this disease?

Reply:
Good point! I'll add these detailed information. Thanks Serena